uaclient/cli.py

```diff
@@ -204,20 +204,20 @@ def new_f(cmd_args, *args, **kwargs):
     return new_f
 
 
-def assert_attached(unattached_msg_tmpl=None):
+def assert_attached(msg_function=None):
     """Decorator asserting attached config.
-
-    :param unattached_msg_tmpl: Optional msg template to format if raising an
-        UnattachedError
+    :param msg_function: Optional function to generate a custom message
+    if raising an UnattachedError
     """
 
     def wrapper(f):
         @wraps(f)
         def new_f(args, cfg, **kwargs):
             if not cfg.is_attached:
-                if unattached_msg_tmpl:
-                    names = getattr(args, "service", "None")
-                    msg = unattached_msg_tmpl.format(name=", ".join(names))
+                if msg_function:
+                    command = getattr(args, "command", "None")
+                    service_names = getattr(args, "service", "None")
```

Comment:
Any reason not to use empty strings as default values ?

Reply:
I just followed what was already there hahaha
But no, we don't even expect those to be absent at all.
Will change for consistency sake

Reply:
done